mlir/lib/Dialect/MIOpen/Generator/Conv2dGenerator.cpp

```diff
@@ -121,8 +136,10 @@ LogicalResult Conv2dGenerator::parseConvConfig(const char *arguments) {
     strToInt("dilation_w", config.dilationWidth);
     strToInt("conv_stride_h", config.strideHeight);
     strToInt("conv_stride_w", config.strideWidth);
-    strToInt("padding_h", config.paddingHeight);
-    strToInt("padding_w", config.paddingWidth);
+    strToInt("padding_h", config.paddingHeightLeft);
+    strToInt("padding_h", config.paddingHeightRight);
+    strToInt("padding_w", config.paddingWidthLeft);
+    strToInt("padding_w", config.paddingWidthRight);
```

Comment:
This looks right. It will use padding height from client to store same values to left and right padding. So it did the same for width.

In the future when we do see the need for client to give 4 padding values, we can force the client to set all 4 values.